### PR TITLE
40 - List image

### PR DESCRIPTION
**Pull request by kelepirci**
List all images.

### CODE REVIEW

File: image/views.py [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+    title = _("Manage Images")
+    subtitle = _("create,list and manage images")
+    user_profile = request.user.profile
+    conn = ConnectDocker(user_profile.selected_site)
+    client = conn.docker_connect()
+    images = client.images.list(all=True)
+
+
+    return render(request, "image/image_list.html", {"images": images,
+                                                             "title": title,
```
Comment (sametbalta) [inline image/views.py:28]: There is an indentation fault but it is not a big deal

File: templates/image/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Manage Images</title>
```
Comment (sametbalta) [inline templates/image/index.html]: Missing "trans" tag

File: templates/image/index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+</head>
+<body>
+  <p>Manage images</p>
```
Comment (sametbalta) [inline templates/image/index.html]: Missing "trans" tag

File: templates/network/index.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Network</title>
```
Comment (sametbalta) [inline templates/network/index.html]: Missing "trans" tag

File: templates/volume/index.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Volume</title>
```
Comment (sametbalta) [inline templates/volume/index.html:5]: Missing "trans" tag

File: volume/views.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    title = _("Manage Volume")
+    subtitle = _("create,list and manage volumes")
+    user_profile = request.user.profile
+    conn = ConnectDocker(user_profile.selected_site)
+    client = conn.docker_connect_api()
+    volumes = client.volumes()
+
+
+    return render(request, "volume/volume_list.html", {"volumes": volumes,
+                                                         "title": title,
```
Comment (sametbalta) [inline volume/views.py]: Wrong indentation

**Review by sametbalta — COMMENTED** [review]
Good work! Thank you very much. I think it is OK generally.

**Review by kelepirci — COMMENTED** [review]
Samet, can you please review the changes?

**Review by sametbalta — APPROVED** [review]

**Pull request merged**